public/v2/apps/discourse.json

```diff
@@ -0,0 +1,123 @@
+{
+    "captainVersion": "2",
+    "documentation": "Taken from",
+    "displayName": "discourse",
+    "description": "",
+    "dockerCompose": {
+        "version": "2",
+        "services": {
+          "$$cap_appname-postgresql": {
+            "documentation": "Taken from https://hub.docker.com/r/bitnami/postgresql",
+            "image": "docker.io/bitnami/postgresql:$$cap_postgresql_version",
+            "notExposeAsWebApp": "true",
+            "restart": "always",
+            "environment": [
+              "ALLOW_EMPTY_PASSWORD=yes"
+            ],
+            "volumes": [
+              "postgresql-data:/bitnami/postgresql"
+            ]
+          },
+          "$$cap_appname-redis": {
+            "documentation": "Taken from https://hub.docker.com/r/bitnami/redis",
+            "image": "docker.io/bitnami/redis:$$cap_redis_version",
+            "notExposeAsWebApp": "true",
+            "restart": "always",
+            "environment": [
+              "ALLOW_EMPTY_PASSWORD=yes"
+            ],
+            "volumes": [
+              "redis-data:/bitnami"
+            ]
+          },
+          "$$cap_appname-discourse": {
+            "documentation":"Taken from https://hub.docker.com/r/bitnami/discourse",
+            "image": "docker.io/bitnami/discourse:$$cap_disource_version",
+            "containerHttpPort": "3000",
+            "depends_on": [
+              "$$cap_appname-postgresql",
+              "$$cap_appname-redis"
+            ],
+            "volumes": [
+              "discourse-data:/bitnami"
+            ],
+            "environment": [
+              "POSTGRESQL_HOST=postgresql",
+              "POSTGRESQL_ROOT_USER=postgres",
+              "POSTGRESQL_CLIENT_CREATE_DATABASE_NAME=bitnami_application",
+              "POSTGRESQL_CLIENT_CREATE_DATABASE_USERNAME=bn_discourse",
+              "POSTGRESQL_CLIENT_CREATE_DATABASE_PASSWORD=bitnami1",
+              "DISCOURSE_POSTGRESQL_NAME=bitnami_application",
+              "DISCOURSE_POSTGRESQL_USERNAME=bn_discourse",
+              "DISCOURSE_POSTGRESQL_PASSWORD=bitnami1",
+              "DISCOURSE_HOSTNAME=$$cap_appname.$$cap_root_domain"
+            ]
+          },
+          "$$cap_appname-sidekiq": {
+            "documentaion": "Taken from https://hub.docker.com/r/bitnami/discourse",
+            "image": "docker.io/bitnami/discourse:$$cap_discourse_version",
+            "notExposeAsWebApp": "true",
+            "depends_on": [
+              "$$cap_appname-discourse"
+            ],
+            "volumes": [
+              "discourse-data:/bitnami"
+            ],
+            "command": "nami start --foreground discourse-sidekiq",
```

Comment:
Command is not respected in one-click apps. You can simply change this sidekiq to use an image:
```
FROM docker.io/bitnami/discourse:$$cap_discourse_version
CMD ["nami" , "start" , "--foreground discourse-sidekiq"]
```

Reply:
So can I just put the image on Docker Hub then? Once I make it of course.

Reply:
No - you don't need to do that. Look at this example:
https://github.com/caprover/one-click-apps/blob/37abf28c342eee9dab8054fa4d4bb1862c2958c4/public/v2/apps/chatwoot.json#L23-L26

Reply:
Ahh, I see, thank you.